canonical-web-and-design/ubuntu.com
# Make /getubuntu redirects more greedy

## Done

- Make /getubuntu redirects more greedy
- Looking at 404's people are also going to sub-pages of getubuntu

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/getubuntu/download
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are sent to /download
